src/iotile/models/properties.ts

```diff
@@ -0,0 +1,8 @@
+export class Properties {
+
+  constructor(datas) {
+    for (let data of datas) {
+      this[data.name] = data.value;
```

Comment:
I don't like this. Properties should have a real field to store the MAP.
It should then have proper methods to get the data e.g.

```
properties.set('foo', 'bar')
properties.get('foo')
```

This will allow you to add business logic and error checking